docs/advanced/storagenetwork.md

```diff
@@ -8,440 +8,427 @@ title: "Storage Network"
+          inet 10.0.16.14/20 brd 10.0.31.255 scope global lhnet1
+      ...
+
+      $ ip route
+      default via 169.254.1.1 dev eth0
+      10.0.16.0/20 dev lhnet1 proto kernel scope link src 10.0.16.14
+      169.254.1.1 dev eth0 scope link
+      ```
+
+      :::note
+
+      The storage network link always inherits the MTU value from the attached [cluster network](../networking/clusternetwork.md#cluster-network), regardless of the [configured MTU value](../networking/clusternetwork.md#change-the-mtu-of-a-network-configuration-with-an-attached-storage-network).
+
+      :::
+
+    1. Start a simple HTTP server in one pod.
+    
+      You must explicitly bind this HTTP server to the storage network IP.
 
-Explicitly bind the simple HTTP server to the storage network IP.
+      Example:
 
-:::
+      ```
+      $ python3 -m http.server 8000 --bind 10.0.16.14 (replace with your pod storage network IP)
+      ```
 
-4.4 Test the HTTP server in another pod.
+    1. Test the HTTP server in another pod.
 
-Example:
+      Example:
 
-```
-From instance-manager-r-43f1624d14076e1d95cd72371f0316e2 (IP 10.0.16.8)
+      ```
+      From instance-manager-r-43f1624d14076e1d95cd72371f0316e2 (IP 10.0.16.8)
 
-$ curl http://10.0.16.14:8000
-```
+      $ curl http://10.0.16.14:8000
+      ```
 
-When the storage network is functioning correctly, the `curl` command returns a list of files on the HTTP server.
+      When the storage network is functioning correctly, the `curl` command returns a list of files on the HTTP server.
 
-4.5 (Optional) Troubleshoot issues.
+    1. (Optional) Troubleshoot issues.
 
-The storage network may malfunction because of issues with the external network, such as the following:
+      The storage network may malfunction because of issues with the external network, such as the following:
 
-- Physical NICs (installed on Harvester nodes) that are associated with the storage network were not added to the same VLAN in the external switches.
+      - Physical NICs (installed on Harvester nodes) that are associated with the storage network were not added to the same VLAN in the external switches.
+      - The external switches are not correctly connected and configured.
 
-- The external switches are not correctly connected and configured.
+Once the configuration is verified, you can manually start virtual machines when necessary.
 
+## Best Practices
 
-### Start VM Manually
+- When configuring an [IP range](#configuration-example) for the storage network, ensure that the allocated IP addresses can service the future needs of the cluster. This is important because Longhorn pods (`instance-manager` and `backing-image-manager`) stop running when new nodes are added to the cluster or more disks are added to a node after the storage network is configured, and when the required number of IPs exceeds the allocated IPs. Resolving the issue involves reconfiguring the storage network with the correct IP range.
```

Comment:
```suggestion
- When configuring an [IP range](#prerequisites) for the storage network, ensure that the allocated IP addresses can service the future needs of the cluster. This is important because Longhorn pods (`instance-manager` and `backing-image-manager`) stop running when new nodes are added to the cluster or more disks are added to a node after the storage network is configured, and when the required number of IPs exceeds the allocated IPs. Resolving the issue involves reconfiguring the storage network with the correct IP range.
```